id/index.html

```diff
@@ -362,8 +362,8 @@ <h2 class="myth__title">Aksesibilitas hanya dapat diuji oleh penyandang disabili
           Dibuat dengan
           <a target="_blank" href="https://en.wikipedia.org/wiki/Hand">Tangan</a>. Di-host di
           <a target="_blank" href="https://www.netlify.com/">Netlify</a>. Kode sumber ada di
-          <a target="_blank" href="https://github.com/sergeicodes/a11ymyths">GitHub</a>.
-          
+          <a target="_blank" href="https://github.com/sergeicodes/a11ymyths">GitHub</a>. Translated by
+          <a target="_blank" href="https://github.com/franzaurus">Fransisca</a>
```

Comment:
@franzaurus could you please translate this to Indonesian? Thanks!

Reply:
Hi @sergeicodes,

I don't have permission to commit to this branch. So I translate it here:
Translated by=Diterjemahkan oleh
My name=Fransisca Hanum

Thanks so much!